### ISSUE TITLE
4-byte private ASN is not displayed properly

### ISSUE THREAD

**Issue by lordofire** [sbezverk/gobmp#183]
Snippet of the json output for `PeerStateChangeMsg`. As you could see, the 4-byte `local_asn` is shown as `-71967296`, whereas the correct output should be unsigned int32 format with `4223000000`
```
{
   "type":10,
   "key":"b7de6bb60d4e6fe9bec37585bce6ea30",
   "value":{
      "action":"add",
      "router_hash":"b7de6bb60d4e6fe9bec37585bce6ea30",
      "remote_bgp_id":"10.3.0.17",
      "router_ip":"10.3.0.1",
      "timestamp":"Dec 20 06:17:33.000036",
      "remote_asn":64520,
      "remote_ip":"10.3.0.17",
      "peer_rd":"0:0",
      "remote_port":59737,
      "local_asn":-71967296,
      "local_ip":"10.3.0.1",
      "local_port":179,
      "local_bgp_id":"172.16.100.8",
      "adv_cap":{
```

Looking forward to the fix on this. Thanks!

**Comment by sbezverk** [comment]
@lordofire thank you for reporting this issue, I built new docker container `docker.io/sbezverk/gobmp:issue_183` with a potential fix, could you please try it and confirm the fix.

**Comment by lordofire** [comment]
Yeah, it works fine now. Thanks for the fix :)

**Comment by lordofire** [comment]
btw, a little bit off topic here, is that possible to upload a v1.0.0 docker image to the docker hub? That would be easier to use compared to latest. Thanks in advance.

**Comment by lordofire** [comment]
Hi Serguei, could you please help build this patch to v1.0.1 as well? Thanks.

**Comment by sbezverk** [comment]
@lordofire it should be included in v1.0.1
